gplog/gplog.go

```diff
@@ -390,7 +391,12 @@ func Fatal(err error, s string, v ...interface{}) {
 	message := ""
 	stackTraceStr := ""
 	if err != nil {
-		message += fmt.Sprintf("%v", err)
+		if _, ok := err.(*pgconn.PgError); !ok {
+			message += fmt.Sprintf("%v", err)
+		} else {
+			message += fmt.Sprintf("%v; ", err.(*pgconn.PgError).Message)
```

Comment:
For Message and Detail I would probably use the `%s` format string rather than `%v`